Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/case-report-list.component.scss

```diff
@@ -52,6 +52,19 @@ th {
     }
 }
 
+.table>tbody>tr>td,
```

Comment:
Is the classname a bit generic? And I would probably only make the selector:
.className td,
.className th

Reply:
See comment below